### PR TITLE
IntelliJ guide, analysis chapter

### PR DESCRIPTION
**Pull request by RichardHoch**
MTA 5.2.1

Partially resolves https://issues.redhat.com/browse/WINDUP-3207

This PR contains the text of the analysis (reviewing and resolving) chapter of the IntelliJ IDEA plugin guide. 

Preview of chapter: https://deploy-preview-518--windup-documentation.netlify.app/docs/intellij-idea-plugin-guide/master/index.html#reviewing-and-resolving-migration-issues

### CODE REVIEW

**Comment by RichardHoch** [comment]
@johnsteele Please review this PR. it is the last PR for the guide.

**Review by johnsteele — APPROVED** [review]

**Review by sshveta — APPROVED** [review]

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-reviewing-issues_{context}"]
+= Reviewing issues
+
+You can use the {ProductShortName} plugin icons to prioritize issues based on their severity. You can  see which issues have a _Quick Fix_ automatic code replacement and which do not.
+
+The results of an analysis are displayed in a directory format, showing the _hints_ and _classifications_ for each application analyzed.
+
+A hint is a read-only snippet of code that contains a single issue that either must or should be addressed before an application can modernized or migrated. Often a suggested Quick Fix code replacement is suggested. You can accept or ignore any suggested Quick Fixes.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-reviewing-issues.adoc:12]: Lot of passive voice in this sentence.

```suggestion
A _hint_ is a read-only snippet of code that contains a single issue that you should or must address before you can  modernize or migrate an application. Often the plugin suggests a Quick Fix, which you can either accept or ignore.
```

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-reviewing-issues_{context}"]
+= Reviewing issues
+
+You can use the {ProductShortName} plugin icons to prioritize issues based on their severity. You can  see which issues have a _Quick Fix_ automatic code replacement and which do not.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-reviewing-issues.adoc:8]: Seems like _Quick Fix_ means "automatic code replacement". If so, then define it here, and just use "Quick Fix" subsequently, without "automatic code replacement".

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+The results of an analysis are displayed in a directory format, showing the _hints_ and _classifications_ for each application analyzed.
+
+A hint is a read-only snippet of code that contains a single issue that either must or should be addressed before an application can modernized or migrated. Often a suggested Quick Fix code replacement is suggested. You can accept or ignore any suggested Quick Fixes.
+
+A classification is a file that has an issue but does not have any suggested Quick Fixes. You can edit a classification.
+
+.Procedure
+
+. In the {ProjectName} view, select a run configuration directory on the left pane.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-reviewing-issues.adoc:18]: on -> in

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
++
+The *Report* and *Results* icons are displayed.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-reviewing-issues.adoc:20]: Are they icons or buttons?

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+. Click the *Results* icon.
++
+The modules and applications of the run configuration are displayed. Below each application, the *Hints* and *Classifications* icons are displayed.
+
+. Prioritize issues based on the following icons, which are displayed next to each hint:
+
+** image:intellij-idea-mandatory.png[Mandatory] : The issue must be fixed for a successful migration.
+** image:intellij-idea-potential.png[Warning] : The issue might need to be addressed during migration.
+
+. Optional: To learn more about a hint, right-click on it and select *Show More Details*.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-reviewing-issues.adoc:30]: Style guideline: You click something, you don't click _on_ something.

So "right-click it". Search for other instances of "click on" as well.

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc [hunk 6]
```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-resolving-issues_{context}"]
+= Resolving issues
+
+You can resolve issues by doing one of the following:
+
+* Using a _Quick Fix_ automatic code replacement to fix a code snippet hint has a hint
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-resolving-issues.adoc:10]:
```suggestion
* Using a _Quick Fix_ to fix a code snippet that has a hint
```

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+* Editing the code of a file that appears in a a classification
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-resolving-issues.adoc:11]:
```suggestion
* Editing the code of a file that appears in a classification
```

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+. In the {ProjectName} view, select a run configuration directory on the left pane.
++
+The *Report* and *Results* icons are displayed.
+. Click the *Results* icon.
++
+The modules and applications of the run configuration are displayed. Below each application, the *Hints* and *Classifications* icons are displayed.
+
+. Prioritize issues based on the following icons, which are displayed next to each hint:
+
+** image:intellij-idea-mandatory.png[Mandatory] : The issue must be fixed for a successful migration.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-reviewing-issues.adoc:27]: passive voice here an the next in the item.

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+. In the left pane, click on a hint that has an error indicator.
++
+Any Quick Fixes are displayed as child folders with the Quick Fix icon (image:intellij-idea-info.png[Quickfix]) on their left side.
+. Right-click a Quick Fix and select *Preview Quick Fix*.
++
+The current code and the suggested change are displayed in the *Preview Quick Fix* window.
+. To accept the suggested fix, click *Apply Quick Fix*.
+. Optional: Right-click on the issue and select *Mark As Complete*.
++
+A green check appears by the hint, replacing the error indicator.
```
Comment (stoobie) [inline docs/topics/intellij-idea-plugin-resolving-issues.adoc:28]: Do you have an icon to put here?

**Review by stoobie — CHANGES_REQUESTED** [review]

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 1 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc:12]: I've incorporated your recommendations except for in the last sentence. You suggest using an active verb with the system, which violates the rule against anthropomorphism.

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 2 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc:8]: Done.

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 3 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc:18]: Done.

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 5 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc:30]: Done.

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 4 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc:20]: icons.

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc (hunk 6 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-resolving-issues.adoc:10]: Fixed.

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc (hunk 7 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-resolving-issues.adoc:11]: Fixed.

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc (hunk 9 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-resolving-issues.adoc:28]: Added.

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 8 above)
Comment (RichardHoch) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc:27]: Fixed.

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 3 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc]: LGTM

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 4 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc]: LGTM

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 5 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc]: lgtm

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc (hunk 6 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-resolving-issues.adoc]: lgtm

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc (hunk 7 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-resolving-issues.adoc]: lgtm

File: docs/topics/intellij-idea-plugin-reviewing-issues.adoc (hunk 8 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-reviewing-issues.adoc]: lgtm

File: docs/topics/intellij-idea-plugin-resolving-issues.adoc (hunk 9 above)
Comment (stoobie) [reply docs/topics/intellij-idea-plugin-resolving-issues.adoc]: lgtm

**Review by stoobie — APPROVED** [review]
LGTM

**Comment by stoobie** [comment]
@RichardHoch Do you want to merge?

**Pull request merged**